test/k8s-e2e/convtojson.py

```diff
@@ -105,20 +108,22 @@ def help(msg=HELPMESSAGE):
             targetdir = arg
         elif opt in ("-t", "--timestamp"):
             execution_timestamp = arg
+        elif opt in ("-f", "--descfile"):
```

Comment:
Maybe `-f` is confusing when talking about inputfile and descfile.
I would suggest `-d` or `--desc`.